pyaedt/maxwell.py

```diff
@@ -293,6 +293,12 @@ def assign_matrix(
         >>> selection = ['Current1', 'Current2', 'Current3', 'Current4']
         >>> turns = [5, 1, 2, 3]
         >>> L = m2d.assign_matrix(sources=selection, matrix_name="Test2", turns=turns, group_sources=group_sources)
+
+        Set matrix in a Maxwell DC Conduction analysis.
```

Comment:
@IreneWoyna good example of docstring improvement!
If you want to add an example in the docstring you have to specify it as: 
`
        Examples
        ----------
        Set matrix in a Maxwell DC Conduction analysis..

        >>> m2d.assign_voltage(["Port1"], amplitude=1, name="1V")
        >>> m2d.assign_voltage(["Port2"], amplitude=0, name="0V")
        >>> m2d.assign_matrix(sources=['1V'], group_sources=['0V'], matrix_name="Matrix1")
`